index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Architecture Bootcamp</title>
+  <link href="./styles.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet"> 
+</head>
+<body>
+  <header class="latto">
+    <div id="desktop-toolbar">
+      <a href="#"><img src="./icons/icon-facebook-white.png" alt="Link to Facebook"></a>
+      <a href="#"><img src="./icons/icon-twitter-white.png" alt="Link to Twitter"></a>
+      <select name="language" id="language">
+        <option value="english">English</option>
+        <option value="uzbek">Uzbek</option>
+        <option value="russian">Russian</option>
+      </select>
+      <a href="#">My Page</a>
+      <a href="#">Log in</a>
+    </div>
+    <nav>
+      <a id="menu-btn" href="#"><img src="./icons/icon-menu-grey-50x60.png" alt="Button to open mobile menu"></a>
+      <a class="logo" href="./index.html"><img src="./icons/logo6.png" alt="Logo and link to the homepage"></a>
+      <ul class="desktop-navigation">
+        <li><a href="./about.html">About</a></li>
+          <li><a href="./program.html">Program</a></li>
+          <li><a href="./join.html">Join</a></li>
+          <li><a href="./sponsor.html">Sponsor</a></li>
+          <li><a href="./news.html">News</a></li>
+      </ul>
+      <a href="#" class="campaign">TIACE Campaign</a>
+    </nav>
+    <div id="mobile-menu" class="mobile-menu hidden latto">
+      <a id="close-menu" href="#"><img src="./icons/cancel-grey.png" alt="Button to close mobile menu"></a>
+      <ul>
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="./about.html">About</a></li>
+        <li><a href="./program.html">Program</a></li>
+        <li><a href="./join.html">Join</a></li>
+        <li><a href="./sponsor.html">Sponsor</a></li>
+        <li><a href="./news.html">News</a></li>
+      </ul>
+    </div>
+  </header>
+  <main class="latto">
+    <section id="headline">
+      <q class="coral subtitle">Create our shining future!</q>
+      <!-- <p class="coral subtitle">Tashkent Institute of Architecture and Civil Engineering</p> -->
+      <h1>YOUNG ARCHITECTS BOOTCAMP<br>
+        2023</h1>
+      <p class="annotation">The Architecture Boot Camp is a year-long program built to provide 
+        students with a platform to discuss, analyze, and interpret the impact 
+        of design, art, and architecture in the built environment through 
+        a focused curriculum and creative outcomes.</p>
+      <p class="date">2023.05.12 ~ 2023.08.12</p>
+      <p class="address">@ TIACE, 13, Navoi str, Tashkent</p>
+    </section>
+    <section id="program" class="program">
```

Comment:
I would recommend adding a border effect upon hovering over each article just like in the provided design.
As you can see, when the user hovers over a card, a white border appears around the card.
![image](https://github.com/Zilola-Nazarova/capstone-project-module-1/assets/33268927/45336072-782d-4e38-aad4-c37328452638)